content/en/docs/release-oversight/test-case-analyzer.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Test Case Analyzer"
+description: Introduction of a new tool to analyze test cases across multiple job runs.
+---
+
+### Summary
+On top of the existing aggregator framework, the new Test Case Analyzer tool is introduced to analyze specific test results from a much broader set of candidate job runs, without requiring the entire job to pass.
+
```

Comment:
It sounds like we're choosing some informing tests to track so that we can get some signal.  Where are those tests configured?

And when the Test Case Analyzer is put into place, will we see a separate section for the results in the same release-controller page we use to monitor nightlies?

Reply:
The jobs that will be analyzed is the list from the release page we monitor. We choose a subset from that list based on variants. 

Yes, this test case analyzer will appears as a new job in that list.